Documentation/ConfigureAppSecrets.md

```diff
@@ -0,0 +1,27 @@
+# Configure App Secrets
```

Comment:
This sections is mostly copy-pasted from the root README, with a few minor changes.